lib/park.rb

```diff
@@ -0,0 +1,25 @@
+class Park
+  attr_reader :name, :admission_price, :vehicles, :passengers, :revenue
+  def initialize(name, admission_price)
+    @name = name
+    @admission_price = admission_price
+    @vehicles = []
+    @revenue_total = 0
+  end
+
+  def add_vehicle(vehicle)
+    @vehicles << vehicle
+  end
+
+  def passengers
+    @vehicles.map do |vehicle|
+      vehicle.passengers 
+    end
```

Comment:
```suggestion
    @vehicles.map do |vehicle|
      vehicle.passengers 
    end.flatten
```

This is really close. `vehicle.passengers` returns that vehicle's array of passengers which means this `map` as a whole will return an array of arrays. If you call `flatten` on it, it will flatten all of those nested arrays so now you only have one outer array containing all the objects that were previously nested inside other arrays.